sdk/communication/AzureCommunication/Source/CommunicationCloudEnvironment.swift

```diff
@@ -39,8 +39,8 @@ import Foundation
         self.environmentValue = environmentValue
     }
 
-    static func fromModel(environmentModel: CommunicationCloudEnvironmentModel) -> CommunicationCloudEnvironment {
-        return CommunicationCloudEnvironment(environmentValue: environmentModel.requestString)
+    public func getEnvironmentValue() -> String {
+        return environmentValue
     }
```

Comment:
It looks like the removal of `fromModel(...)` is a breaking change and should be annotated in the changelog.